src/app/intro/views/UserIntro/UserIntro.test.tsx

```diff
@@ -123,52 +105,42 @@ describe("UserIntro", () => {
   });
 
   it("marks the intro as completed when clicking the continue button", async () => {
-    const store = mockStore(state);
     renderWithProviders(<UserIntro />, {
       initialEntries: ["/intro/user"],
-      store,
     });
 
     await waitForLoading();
     await userEvent.click(
       screen.getByRole("button", { name: UserIntroLabels.Continue })
     );
-    expect(
-      store
-        .getActions()
-        .some((action) => action.type === userActions.markIntroComplete().type)
-    );
```

Comment:
This test doesn't assert anything anymore - is there not an endpoint that gets called when this button is clicked?